src/sass/components/_footer.scss

```diff
@@ -80,3 +82,30 @@ footer {
     }
   }
 }
+
+@include media-breakpoint-down(sm) {
+
+body{
+  footer {
+    .footer-menu{
+
+        text-align: center;
+
+    }
+    .bottom-bar {
+      position: relative;
+      .unidentify-object {
+        position: absolute;
+        bottom: 90%;
+        left: 0;
+        width: 100%;
+      }
+      .copyright {
+        p {
```

Comment:
again, it's not necessary to have so complicated selector here, as you'd easily achieve the same on root level with
```
.copyright  {
    p {
        text-align...
    }
}
```